### DISCUSSION TITLE
Hacked

### DISCUSSION THREAD

**Discussion by Neil187** [community/community#146638] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

What can I do about a family member that is able to control everything in my phone, I tried changing my passwords, phone numbers, phones, but nothing works and nobody will help me, they even put an encryption on my ID.me, need help what do I do

**Comment by MindfulLearner** [comment]
Initialize your phone first. 

About 
>  I tried changing my passwords, phone numbers, phones, but nothing works and nobody will help me,

You can't do nothing man! If they have parental controll, emails password yo ucant do nothing about it. sorry.
